.github/workflows/odh-build-and-publish-operator-image.yaml

```diff
@@ -0,0 +1,106 @@
+# This is a copy of the publish-core-images.yaml and has been customized to
+# use the quay login credentials.
+# The unused parts of the original have been commented out on purpose.
+name: ODH -- Build and Publish Training Operator Images
+
+on:
+  - push
+  - pull_request
+
+jobs:
+    build-and-publish-opeator:
+      name: ODH - Build and (or) Publish Image
+      runs-on: ubuntu-latest  
+      env:
+        GOPATH: ${{ github.workspace }}/go
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          
+        - name: Set up Go
+          uses: actions/setup-go@v3
+          with:
+            go-version-file: go.mod
+      
+        - name: Login to Quay.io
+          id: podman-login-quay
+          # Trigger step only for specific branch (master, v.*-branch) or tag (v.*).
+          if: >-
+            (github.ref == 'refs/heads/dev' || (startsWith(github.ref, 'refs/heads/v') && endsWith(github.ref, '-branch')) || startsWith(github.ref, 'refs/tags/v')) 
+          shell: bash
+          run: |
+              podman login --username ${{ secrets.QUAY_USERNAME }} --password ${{ secrets.QUAY_TOKEN }} quay.io
+        
+        - name: Run go mod
+          shell: bash
+          run: |
+            go mod download
+        
+        - name: Add docker tags
+          id: meta
+          uses: docker/metadata-action@v5
+          with:
+            images: quay.io/${{ vars.QUAY_REPO_NAME }}/training-operator
+            tags: |
+              type=raw,latest
+              type=sha,prefix=v1-  
+        
+        # Build operators inside the gh runner vm directly and then copy the go binaries to docker images using the Dockerfile.buildx
+        - name: Build linux/amd64 operator binary
+          env:
+            CGO_ENABLED: 1
+            GOOS: linux
+            GOARCH: amd64
+          shell: bash
+          run: |
+            CGO_ENABLED=$CGO_ENABLED GOOS=$GOOS GOARCH=$GOARCH go build -tags strictfipsruntime -a -o manager-$GOARCH cmd/training-operator.v1/main.go
+   
+        - name: Build linux/arm64 operator binary
+          env:
+            CC: aarch64-linux-gnu-gcc
+            CGO_ENABLED: 1
+            GOOS: linux
+            GOARCH: arm64
+          shell: bash
+          run: |
+            sudo apt-get update
+            sudo apt-get install -y gcc-aarch64-linux-gnu libc6-dev-arm64-cross
+            CC=$CC CGO_ENABLED=$CGO_ENABLED GOOS=$GOOS GOARCH=$GOARCH go build -tags strictfipsruntime -a -o manager-$GOARCH cmd/training-operator.v1/main.go
```

Comment:
still need `"CC=$CC CGO_ENABLED=$CGO_ENABLED GOOS=$GOOS GOARCH=$GOARCH" `if already set in `env`

Reply:
I am not sure if I understand the comment, is it a syntax issue ?

Reply:
I understood. fixed in [10e7ab9](https://github.com/opendatahub-io/training-operator/pull/4/commits/10e7ab9d1a25e4a88ce167c09935f833a06d7208)